client/src/pages/admin/registrations.tsx

```diff
@@ -112,11 +145,11 @@ class RegistrationsPage extends React.Component<Props, State> {
   };
 
   approve = (e: any) => {
-    this.handleSubmit(e, 'approve');
+    this.handleSubmit(e, 'approved');
```

Comment:
i believe you also need to updated the string on the server side. 
https://github.com/rolling-scopes/rsschool-app/blob/master/server/src/routes/registry.ts#L132